src/components/car/car_form.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import './car_form.css';
+
+interface Car {
+  id: number;
+  maker: string;
+  model: string;
+  year: number;
+}
+
+const initialCarState: Car = {
+  id: 0,
+  maker: 'Toyota',
+  model: 'Toyota RQ',
+  year: 2000,
+};
+
+// interface CarFormProps {
+//   cars: Car[];
+//   setCars: (cars: Car[]) => void;
+//   addCar: (car: Car) => void;
+// }
```

Comment:
That was a good idea to declare a interface for properties that you component can accept.

Reply:
fix